src/ngx_http_lua_worker.c

```diff
@@ -161,6 +163,15 @@ ngx_http_lua_ffi_get_process_type(void)
     }
 #endif
 
+    if (ngx_process == NGX_PROCESS_SINGLE) {
+        ccf = (ngx_core_conf_t *) ngx_get_conf(ngx_cycle->conf_ctx,
+                                               ngx_core_module);
+
+        if (ccf->master) {
+            return NGX_PROCESS_MASTER;
```

Comment:
Better drop a note to the documentation for this? (If there's none)

Reply:
@agentzh 
Updated.